packages/token-bridge-sdk/src/hooks/useTransactions.ts

```diff
@@ -342,6 +342,9 @@ const useTransactions = (): [Transaction[], TransactionActions] => {
     // It's ok to bail here, as the RetryableTxnsIncluder will pick it up
     const res = await ethDepositMessage.wait(undefined, 500)
 
+    // TODO: Do we need to restructure this to return EthDepositStatus instead?
```

Comment:
@spsjvc @yahgwai question linked to https://github.com/OffchainLabs/arbitrum-sdk/pull/206